google-cloud-sdk/src/com/google/cloud/tools/intellij/appengine/java/sdk/CloudSdkValidator.java

```diff
@@ -58,8 +58,16 @@ public static CloudSdkValidator getInstance() {
       // If the Cloud SDK is not found, don't bother checking anything else
       return validationResults;
     } catch (CloudSdkOutOfDateException exception) {
-      validationResults.add(CloudSdkValidationResult.CLOUD_SDK_VERSION_NOT_SUPPORTED);
-    } catch (CloudSdkVersionFileException e) {
+      validationResults.add(CloudSdkValidationResult.CLOUD_SDK_NOT_MINIMUM_VERSION);
+    } catch (CloudSdkVersionFileException exception) {
+      // TODO once appengine-plugins core change is merge in, this exception will not be thrown
```

Comment:
I need to catch this now, but once we update the core library we can remove this `catch` block